openapi.yaml

```diff
@@ -0,0 +1,734 @@
+        example: https://bethesda.net/
+  DeveloperResult:
+    properties: 
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        $ref: '#/definitions/DeveloperResource'
+  DeveloperResults:
+    properties: 
+      links:
+        allOf:
+          - $ref: '#/definitions/SelfLink'
+          - $ref: '#/definitions/PaginationLinks'
+      meta:
+        $ref: '#/definitions/Meta'
+      data:
+        type: array
+        items:
+          $ref: '#/definitions/DeveloperResource'
+  DeveloperPostBody:
+    properties:
+      data:
+        type: object
+        properties:
+          name:
+            $ref: '#/definitions/DeveloperResource/properties/name'
+          website:
+            $ref: '#/definitions/DeveloperResource/properties/website'
+        required:
+          - name
+          - website
+  DeveloperPatchBody:
+    properties:
+      data:
+        type: object
+        properties:
+          name:
+            $ref: '#/definitions/DeveloperResource/properties/name'
+          website:
+            $ref: '#/definitions/DeveloperResource/properties/website'
+  ReviewResource:
+    properties:
+      id:
+        type: string
+      gameId:
+        $ref: '#/definitions/GameResource/properties/id'
+      reviewer:
+        type: string
+        description: Person who wrote the review.
+        example: Commander Shepard
+      score:
+        type: number
+        format: float
+        description: Score for this review out of 5
+        example: 4.5
+      reviewText:
+        type: string
+        description: Text containing the review for the referenced game
+        example: My name is Commander Shepard and this is my favorite game on the citadel.
+      reviewDate:
+        type: string
+        format: date
+        description: Date the review was finalized
+        example: 10/15/2006
+  ReviewResult:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        $ref: '#/definitions/ReviewResource'
+  ReviewResults:
+    properties: 
+      links:
+        allOf:
+          - $ref: '#/definitions/SelfLink'
+          - $ref: '#/definitions/PaginationLinks'
+      meta:
+        $ref: '#/definitions/Meta'
+      data:
+        type: array
+        items:
+          $ref: '#/definitions/ReviewResource'
+  ReviewPostBody:
+    properties: 
+      data:
+        type: object
+        properties:
+          gameId:
+            $ref: '#/definitions/ReviewResource/properties/gameId'
+          reviewText:
+            $ref: '#/definitions/ReviewResource/properties/reviewText'
+          score:
+            $ref: '#/definitions/ReviewResource/properties/score'
+          reviewer:
+            $ref: '#/definitions/ReviewResource/properties/reviewer'
+        required:
+        - gameId
+        - reviewText
+        - score
+        - reviewer
+  ReviewPatchBody:
+    properties: 
+      data:
+        type: object
+        properties:
+          gameId:
```

Comment:
Do we wanna allow users to patch a review's `gameId`? I think it'll make more sense to delete the old review and add a new one for another game. What do you think?

Reply:
The other case that this should be changed is if the chose the wrong game initially. Recreating it would be a good idea in this case.